docs/src/pages/configurations/custom-webpack-config/index.md

```diff
@@ -5,9 +5,8 @@ title: 'Custom Webpack Config'
 
 ## Default mode
 
-The default Webpack config of Storybook is balanced for a medium-size project (specially created with [Create React App](https://github.com/facebookincubator/create-react-app)) or a library. But if you already have your own Webpack setup, that's not useable.
```

Comment:
I know the existing doesn't make much sense anymore but we should still keep some of the same ideas presented here.

Something about the default being a sensible starting point for most projects and frameworks.

Reply:
agreed

Reply:
@langri-sha could you re-add something along these lines, do you think?